.rubocop.yml

```diff
@@ -1,9 +1,13 @@
 AllCops:
   Exclude:
     - 'lib/winrm/winrm_service.rb'
+    - 'lib/winrm/http/transport.rb'
 
 Style/Encoding:
   Enabled: true
 
 Metrics/LineLength:
   Max: 100
+
+StringLiterals:
+  Enabled: false
```

Comment:
I can get on board with single quotes. I'll remove the `StringLiterals` line from .rubocop.yml but I'm going to ignore the transport.rb file for awhile. I'd like to reorganize it a bit and can make the changes then.